commands/root.go

```diff
@@ -26,8 +26,14 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
-	clustercmd "github.com/hazelcast/hazelcast-commandline-client/commands/cluster"
-	mapcmd "github.com/hazelcast/hazelcast-commandline-client/commands/types/map"
+	clusterCmd "github.com/hazelcast/hazelcast-commandline-client/commands/cluster"
+	listCmd "github.com/hazelcast/hazelcast-commandline-client/commands/types/list"
+	mapCmd "github.com/hazelcast/hazelcast-commandline-client/commands/types/map"
+	multimapCmd "github.com/hazelcast/hazelcast-commandline-client/commands/types/multimap"
+	queueCmd "github.com/hazelcast/hazelcast-commandline-client/commands/types/queue"
+	rmapCmd "github.com/hazelcast/hazelcast-commandline-client/commands/types/replicatedmap"
+	setCmd "github.com/hazelcast/hazelcast-commandline-client/commands/types/set"
+	topicCmd "github.com/hazelcast/hazelcast-commandline-client/commands/types/topic"
```

Comment:
Let's remove all `xCmd` imports, except `clusterCmd` and `mapCmd`.

Reply:
Fixed: [a3c9d21](https://github.com/hazelcast/hazelcast-commandline-client/pull/45/commits/a3c9d213edcf6740fd480352948d34aaa5e57b3d).